Traceback (most recent call last):
  File "main.py", line 19, in <module>
    student = Listener(repo)
  File "C:\Users\VB\Documents\GitHub\TPRO\logic\listener.py", line 13, in __init__
    self.process_events(repo.get_issues_events())
  File "C:\Users\VB\Documents\GitHub\TPRO\logic\listener.py", line 34, in process_events
    EV.process(event)
  File "C:\Users\VB\Documents\GitHub\TPRO\logic\event.py", line 480, in process
    EVENTS[event.type](event)
  File "C:\Users\VB\Documents\GitHub\TPRO\logic\event.py", line 222, in review_requested
    check_files(pull)
  File "C:\Users\VB\Documents\GitHub\TPRO\logic\event.py", line 306, in check_files
    check_code(pull, file)
  File "C:\Users\VB\Documents\GitHub\TPRO\logic\event.py", line 167, in check_code
    for transfer in graph[state]:
KeyError: 'comment_summary'
